src/components/exportToCSV/ExportToCsv.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import CustomButton from "../common/Button/button";
+
+interface Properties {
+  members: object[];
+}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function convertToCSV(data: Record<string, any>[]): string {
```

Comment:
I believe there's a better typing you can use here. The data table being used has a type specified